### PR TITLE
[ONNX] Fix split export in opset13

### PR DESCRIPTION
**Pull request by KsenijaS**
Fix split export in opset13

### CODE REVIEW

File: torch/onnx/symbolic_opset13.py
Comment (jiafatom) [inline torch/onnx/symbolic_opset13.py:73]:
```python
sizes = self.type().sizes()
if sizes is not None and sizes[dim] is not None
```

Comment (KsenijaS) [reply torch/onnx/symbolic_opset13.py:73]: good point. Thanks!

File: torch/onnx/symbolic_opset13.py
Comment (jiafatom) [inline torch/onnx/symbolic_opset13.py:75]: `size = sizes[dim]`

**Review by jiafatom — APPROVED** [review]

File: torch/onnx/symbolic_opset13.py
Comment (BowenBao) [inline torch/onnx/symbolic_opset13.py:73]: nit: can use `_get_tensor_dim_size` from `sym_help`.

**Review by BowenBao — APPROVED** [review]
LGTM, CI failure is not related, and should be resolved in current dev branch merge.
Will merge this PR after the dev branch merge.

**Pull request merged**
